jobs/rocoto/postsnd.sh

```diff
@@ -1,24 +1,28 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-
-###############################################################
-# Source FV3GFS workflow modules
-
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
 if [[ ${status} -ne 0 ]]; then
     exit "${status}"
 fi
 
 export job="postsnd"
-export jobid="${job}.$$"
-
-###############################################################
-# Execute the JJOB
-${HOMEgfs}/jobs/JGFS_ATMOS_POSTSND
-status=$?
-
-
-exit "${status}"
 
+# shellcheck disable=SC2153
+if [[ "${RUN}" == "gefs" ]]; then
+    IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
+    export FHR3 jobid
+    for fhr in "${fhr_list[@]}"; do
+        FHR3=$(printf '%03d' "${fhr}")
+        jobid="${job}_f${FHR3}.$$"
+        echo "add J-script here for GEFS."
+    done
+else
+    export jobid="${job}.$$"
+    ################################################################
+    # Execute the JJOB
+    ${HOMEgfs}/jobs/J${RUN^^}_ATMOS_POSTSND
```

Comment:
We only have a GFS version of this script right now. If we add a GDAS one we'd likely just rename the GFS one to be a GLOBAL one. For now, keep this as a GFS script without the `RUN` setting:
```suggestion
    ${HOMEgfs}/jobs/JGFS_ATMOS_POSTSND
```